.goreleaser.yaml

```diff
@@ -15,8 +16,8 @@ builds:
     goarch:
       - amd64
       - arm64
-  # A build with the playground included.
-  - id: "defradb_playground"
+  # Default build with playground
```

Comment:
question: thought we said 2 builds:
1) slim (no extras)
2) default (everything)